runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8355,10 +8355,21 @@ class INTERPRETER_CLASS
 
 		_sendMethod = (J9Method *)(UDATA)J9OBJECT_U64_LOAD(_currentThread, memberNameObject, _vm->vmtargetOffset);
 
-		if (fromJIT) {
-			J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(_sendMethod);
-			UDATA methodArgCount = romMethod->argCount;
+		J9ROMMethod *romMethod = J9_ROM_METHOD_FROM_RAM_METHOD(_sendMethod);
+		UDATA methodArgCount = romMethod->argCount;
+		if (J9_ARE_NO_BITS_SET(romMethod->modifiers, J9AccStatic)) {
+			j9object_t mhReceiver = ((j9object_t *)_sp)[methodArgCount - 1];
+			if (J9_UNEXPECTED(NULL == mhReceiver)) {
+				if (fromJIT) {
+					/* Restore SP to before popping memberNameObject. */
+					_sp -= 1;
+					buildJITResolveFrame(REGISTER_ARGS);
+				}
+				return THROW_NPE;
+			}
```

Comment:
This section duplicates the `THROW_NPE` section above.  The two can be commoned by adding a `throw_npe:` to the first version of this code and replacing this with `goto throw_npe;`